akka-http-tests/src/test/scala/akka/http/scaladsl/Main.scala

```diff
@@ -1,5 +0,0 @@
-package akka.http.scaladsl
-
-class Main {
-
-}
```

Comment:
oh, good catch

Reply:
Hehe, yeah crazy long life for that one.